docs/release-notes.md

```diff
@@ -20,6 +20,7 @@ nav_order: 9
 ### Bug fixes
 
 - Fix Akamai Ignition base64 decoding on padded payloads
+- Add network dependency to `ignition-mount.service`
```

Comment:
Minor: I guess technically we're not adding a dependency, only a strong ordering. Also these notes are typically written to be more concrete to the end-user. E.g. something like

```suggestion
- Fix network race when phoning home on Equinix
```

?